src/app/modules/main/module.nim

```diff
@@ -338,7 +338,7 @@ method load*[T](
   enabled = true)
   self.view.model().addItem(profileSettingsSectionItem)
   if(activeSectionId == profileSettingsSectionItem.id):
-    activeSection = profileSettingsSectionItem
+    activeSection = self.view.model().getItemBySectionType(SectionType.Chat)
```

Comment:
Hm, I can't get it. If activeSectionId is profile why we set activeSection to SectionType.Chat?